### PR TITLE
Modify analysis function

### PR DESCRIPTION
**Pull request by WanzhuoShi**
I have added the PItime of each trial, could you please check it for me? If there is no problem, I will continue to modify other functions.
@young24

### CODE REVIEW

**Review by young24 — COMMENTED** [review]
Looks good. Have you tested on the real data?

File: @FISHDATA/sayIfLearned.m [hunk 1]
```diff
@@ -4,44 +4,25 @@
...
-        end
-    else
-        extincTime = nan;
-    end
-
+function [h, p, extincTime] = sayIfLearned(obj)
+  obj.ratePerformance();
+  Baseline_trial = obj.Res.PItime(1).Trial;
+  Test_trial = obj.Res.PItime(4).Trial;
+  BasePT = obj.Res.PItime(1).PIfish;
```
Comment (young24) [inline @FISHDATA/sayIfLearned.m:11]: It would be better add an explanation of the variable "BasePT"

**Review by young24 — APPROVED** [review]
Great, we can move to the next stage.

**Comment by WanzhuoShi** [comment]
I an sorry. The name should be BasePI, it is a mistake.

**Comment by young24** [comment]
it's fine. Never mind.

**Pull request merged**